package/python-kiwi-spec-template

```diff
@@ -96,56 +108,72 @@ and cloud systems like Xen, KVM, VMware, EC2 and more.
 %package -n python2-kiwi
 Summary:        KIWI - Appliance Builder Next Generation
 Group:          Development/Languages/Python
-Provides:       python-kiwi = %{version}-%{release}
-Provides:       kiwi-image:tbz
 Provides:       kiwi-image:docker
 Provides:       kiwi-image:iso
-Provides:       kiwi-image:vmx
-Provides:       kiwi-image:pxe
 Provides:       kiwi-image:oem
+Provides:       kiwi-image:pxe
+Provides:       kiwi-image:tbz
+Provides:       kiwi-image:vmx
+Provides:       python-kiwi = %{version}-%{release}
+%if 0%{?fedora} || 0%{?suse_version}
 Recommends:     jing
-Requires:       update-alternatives
+%else
+Requires:       jing
```

Comment:
yes please skip it then. if we run into a validation problem and there is no jing on the system kiwi provides a message to the user that more detailed errors can be obtained if jing would have been installed. imho that's enough information to let a user know what to do. It's a nightmare to require a java based software when it is only an optional thing :) Thanks